README.md

```diff
@@ -95,6 +99,15 @@
 - [ ] Service deployed on a production environment
 - [ ] 2 OSS projects besides source{d} enable lookout
 
+##### Announce Lookout to the world
+- [ ] Prepare talk and announcement at GopherCon
+- [ ] Product webpage with description, videos, and instructions on how to use it
```

Comment:
Hi @campoy would you be so kind to briefly explain your expectations on this?

Reply:
Sure, on the new design there will be a page explaining what lookout is, including a quick screencast of how it works (animation works too), basic installation instructions, etc.

Similar to what Google Cloud does with every product, e.g. cloud.google.com/vision. Notice there's even a little demo widget in there, I'd love to have something like that too.